packager/packager.cc

```diff
@@ -206,7 +210,9 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
     vector<Export> exports_;
 
     // Whether the code in this package is compatible for path-based autoloading.
-    bool strictAutoloaderCompatibility_;
+    bool strictAutoloaderCompatibility_ : 1;
+    // Whether this package should just export everything
+    bool exportAll_ : 1;
```

Comment:
This is a nit, but given that there are not a lot of `PackageInfoImpl` structures (even in Stripe's codebase), and that we have a bunch of space to play with in the existing structure, I think turning these into bitfields is premature optimization.

Reply:
Removed!